conviva/docs/gauge.avg_bitrate.md

```diff
@@ -0,0 +1,8 @@
+---
+title: avg_bitrate
+brief: Real-time/live time series of the average bitrate
```

Comment:
I don't understand what you mean by time series, simple series, label series.  If users will know what this means, then never mind. But usually it's stuff like "the average bitrate" or "the number of concurrent plays" etc.

Reply:
Yes the user should know what time series, simple series and label series are.